style/README.md

```diff
@@ -346,3 +346,12 @@ Testing
   appear in the class.
 
 [Imperative mood]: http://en.wikipedia.org/wiki/Imperative_mood
+
+Legacy Code
```

Comment:
Is there a spot for this other than a new "legacy code" section (deprecated != legacy)?